Gemfile.lock

```diff
@@ -47,7 +47,7 @@ GEM
       multi_json (~> 1.0)
     addressable (2.3.5)
     arel (3.0.3)
-    bcrypt (3.1.7)
+    bcrypt (3.1.10)
```

Comment:
Did you also delete `Gemfile.lock`? Otherwise all the same gems would have been installed again.

This is similar to what I'm talking about: http://makandracards.com/makandra/13885-how-to-update-a-single-gem-conservatively. You want to make the minimum change necessary to get what you want. Updating `bcrypt`, `mongoid`, etc. isn't necessary to get `will_paginate_mongoid` working.

Re. those discussions - yeah I posted this in Slack but should have put it in #118: https://github.com/rails/rails/pull/18992. It's a discussion of why it's still good to commit `Gemfile.lock`. It sold it to me.